Tools/px4airframes/markdownout.py

```diff
@@ -89,7 +89,7 @@ def __init__(self, groups, board, image_path):
                     maintainer = param.GetMaintainer()
                     maintainer_entry = ''
                     if maintainer != '':
-                        maintainer_entry = '<p>Maintainer: %s</p>' % (maintainer)
+                        maintainer_entry = '<p>Maintainer: %s</p>' % (maintainer.replace('<', '&lt;').replace('>', '&gt;'))
```

Comment:
Can you use [html.escape](https://docs.python.org/3.8/library/html.html#html.escape) instead?

Reply:
@hamishwillee Friendly ping!

Reply:
Done. Sorry @bkueng. Simply did not see your response in all the noise.